thirdeye-ui/src/app/components/impact-dashboard/summary/summary.component.tsx

```diff
@@ -181,7 +184,8 @@ export const Summary = ({
                 </div>
                 <div ref={summaryRef}>
                     <div>
-                        In the last <b>{verboseSummaryItems.weeks} weeks</b>,&nbsp;
+                        In the last <b>{verboseSummaryItems.weeks} weeks</b>
```

Comment:
Are we sure `.weeks` value be > 0 ? I think its better to conditionally handle message for weeks value like 0, 1 and >1 . WYDT ?

Reply:
Yes weeks value will always be 4, 13, 21